eox_nelp/stats/templates/tenant_stats.html

```diff
@@ -1,13 +1,8 @@
-<%namespace name='static' file='static_content.html'/>
 <%page args="show_videos, show_courses, show_problems, show_learners, show_instructors, show_certificates"/>
```

Comment:
As I know we have to remove this line due webpack rendering error.
https://github.com/eduNEXT/eox-nelp/blob/master/eox_nelp/stats/templates/tenant_stats.html#L1 
Do you know why the <%page args... don't have any problem.

Reply:
probably the first line also works, I didn't test that but the second line work because the webpack doesn't generates a minify version https://github.com/eduNEXT/eox-nelp/blob/d72bc624f7df82ce8e5955bb744d396704ea1e6e/webpack.common.config.js#L57